catalog/github/webhook.js

```diff
@@ -7,58 +7,106 @@ var request = require('request');
  *  @param {string} username - github username
  *  @param {string} accessToken - github access token
  *  @param {string} events - list of the events the webhook should fire on
+ *  @param {string} endpoint - The endpoint the openwhisk deployed on
  *  @return {object} whisk async
  */
 function main(params) {
-  var username = params.username;
-  var repository = params.repository;
-  var accessToken = params.accessToken;
+    var username = params.username;
+    var repository = params.repository;
+    var accessToken = params.accessToken;
 
```

Comment:
@rabbah ,I'm not very clear, could you please give a little more info, thank you.